README.md

```diff
@@ -123,7 +123,29 @@ This will ensure all event handlers will be registered rather being overruled by
 
 ### useRole
 
-(tbd)
+#### Usage
+
+```html
+<script>
+	import { useFloating, useInteractions, useRole } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
+
+	const role = useRole(floating.context, { role: 'tooltip' });
+
+	const interactions = useInteractions([role]);
+</script>
+
+<button {...interactions.getReferenceProps()}>Reference</button>
+<div {...interactions.getFloatingProps()}>Tooltip</div>
+```
+
+#### Options
```

Comment:
Floating UI calls these "Props", not "Options". Though I think options or parameters fit better. Use your best judgement here.

https://floating-ui.com/docs/useRole#props

Reply:
I'll leave this for now since the entire project uses this terminology. We can handle this in a seperate PR if this is an issue.